NodeJS/__integration__/AddressBookClient.it.ts

```diff
@@ -0,0 +1,111 @@
+    expect(addressBookSearchResponse).toBeDefined()
+    expect(addressBookSearchResponse.content.length).toBeGreaterThan(0)
+    expect(addressBookSearchResponse.number).toEqual(0)
+    expect(addressBookSearchResponse.totalPages).toBeGreaterThan(0)
+  }, 3000)
+
+  test('searchContacts of ACO type should return active ACO contacts', async () => {
+    const searchRequest = { type: 'ACO' }
+    const addressBookSearchResponse = await addressBookClient.searchContacts(
+      searchRequest
+    )
+    console.log('GET searchContacts ', addressBookSearchResponse)
+    expect(addressBookSearchResponse).toBeDefined()
+    expect(addressBookSearchResponse.content.length).toBeGreaterThan(0)
+    expect(addressBookSearchResponse.number).toEqual(0)
+    expect(addressBookSearchResponse.totalPages).toBeGreaterThan(0)
+    for (const contact of addressBookSearchResponse.content) {
+      expect(contact.type).toEqual('ACO')
+    }
+  }, 3000)
+
+  test('searchContacts using query should return corresponding contacts', async () => {
+    const searchRequest = { query: 'agent' }
+    const addressBookSearchResponse = await addressBookClient.searchContacts(
+      searchRequest
+    )
+    console.log('GET searchContacts ', addressBookSearchResponse)
+    expect(addressBookSearchResponse).toBeDefined()
+    expect(addressBookSearchResponse.content.length).toBeGreaterThan(0)
+    expect(addressBookSearchResponse.number).toEqual(0)
+    expect(addressBookSearchResponse.totalPages).toBeGreaterThan(0)
+    for (const contact of addressBookSearchResponse.content) {
+      expect(contact.name).toContain('agent')
+      console.log(contact.name)
+    }
+  }, 3000)
+
+  test('searchContacts sorted by should return all active contacts in expected order', async () => {
+    const searchRequest = { orderBy: 'name', orderDirection: 'DESC' }
+    const addressBookSearchResponse = await addressBookClient.searchContacts(
+      searchRequest
+    )
+    console.log('GET searchContacts ', addressBookSearchResponse)
+    expect(addressBookSearchResponse).toBeDefined()
+    expect(addressBookSearchResponse.content.length).toBeGreaterThan(0)
+    expect(addressBookSearchResponse.number).toEqual(0)
+    expect(addressBookSearchResponse.totalPages).toBeGreaterThan(0)
+    // Collect all contact names and sort in expected order
+    const names: string[] = []
+    let i = 0
+    for (const contact of addressBookSearchResponse.content) {
+      names[i++] = contact.name
+    }
+    names.sort()
+    names.reverse()
+    let j = 0
+    for (const contact of addressBookSearchResponse.content) {
+      expect(contact.name).toEqual(names[j++])
+    }
+  }, 3000)
+
+  test('searchContacts with page number/size query should return corresponding page', async () => {
+    const searchRequest = { page: 2, pageSize: 2 }
+    const addressBookSearchResponse = await addressBookClient.searchContacts(
+      searchRequest
+    )
+    console.log('GET searchContacts ', addressBookSearchResponse)
```

Comment:
strongly recommend to remove all `console.*` from tests

Reply:
I replaced them with 'console.debug()" since w/o these logs it's hard to debug. 
hope this is ok.